src/app/home.component.html

```diff
@@ -63,6 +63,236 @@
+    Accept-Encoding: gzip, deflate
+    User-Agent: Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.1)
+    (blank line)
+  </stache-code-block>
+
+  <stache-code-block
+    languageType="java">
+    public class HelloWorld
+      {
+        public static void main(String[] args) {
+        System.out.println("Hello World!");
+      }
+    }
+  </stache-code-block>
+
+  <stache-code-block
+    languageType="json">
+    {
+      "productNameLong": "Stache 2",
+      "productNameShort": "Stache"
+    }
+  </stache-code-block>
+
+  <stache-code-block
+    languageType="markdown">
+    #Heading
+    ##SubHeading
+    - list item
+    - list item
+    - list item
+  </stache-code-block>
+
+  <stache-code-block
+    languageType="php">
+    $json = json_encode($my_object);
+    $array1 = array("a" => "green", "red", "blue", "red");
+    $array2 = array("b" => "green", "yellow", "red");
+    $result = array_diff($array1, $array2);
+  </stache-code-block>
+
+  <stache-code-block
+    languageType="powershell">
+    Function SayHello([string]$name) {
+      Write-Host "Hello, $name."
+    }
+    $Names = @("Bob", "Alice")
+
+    $Names | ForEach {
+      SayHello $_
+    }
+  </stache-code-block>
+
+  <stache-code-block
+    languageType="python">
+    def median(pool):
+      '''Statistical median to demonstrate doctest.
+      >>> median([2, 9, 9, 7, 9, 2, 4, 5, 8])
+      7
+      '''
+      copy = sorted(pool)
+      size = len(copy)
+      if size % 2 == 1:
+      return copy[(size - 1) / 2]
+      else:
+      return (copy[size/2 - 1] + copy[size/2]) / 2
+    if __name__ == '__main__':
+      import doctest
+      doctest.testmod()
+  </stache-code-block>
+
+  <stache-code-block
+    languageType="sass">
+    @mixin large-text
+      color: #ff0000
+
+    @media (min-width: 600px)
+      h1
+        @include large-text
+
+    =large-text
+      color: #ff0000
+
+    h1
+      +large-text
+  </stache-code-block>
+
+  <stache-code-block
+    languageType="scss">
+    @font-face {
+      font-family: "opensans";
+      src: font-url("opensans.ttf");
+    }
+  </stache-code-block>
+
+  <stache-code-block
+    languageType="sql">
+    SELECT COUNT(*) AS cpt, MAX(t.pos) AS max_pos
+    FROM `my_table`
+    LEFT JOIN `other_table` AS t
+    WHERE `somecol` IS NOT NULL
+    ORDER BY t.other_col DESC
+  </stache-code-block>
+
+  <stache-code-block
+    languageType="swift">
+    let someString = "Some string literal value"
+    var emptyString = ""
+    // String interpolation
+    let multiplier = 3
+    "\(multiplier) times 2.5 is \(Double(multiplier) * 2.5)"
+  </stache-code-block>
+
+  <stache-code-block
+    languageType="typescript">
+    interface SearchFunc {
+      (source: string, subString: string): boolean;
+    }
+  </stache-code-block>
+
+  <stache-code-block
+    languageType="yaml">
+    ---
+    layout: layout-base
+    order: 0
+    name: Name of Page
+    description: This is a description
+    ---
+  </stache-code-block>
+
```

Comment:
All these code block samples seems a bit... excessive and cluttering of the home page.   Wouldn't a list of supported language types under the code block component suffice?